### PR TITLE
Update interaction-fundamentals.md

### PR DESCRIPTION
**Pull request by rwinj**

### CODE REVIEW

File: mixed-reality-docs/interaction-fundamentals.md [hunk 1]
```diff
@@ -6,36 +6,62 @@ ms.author: shengkait
...
+keywords: Mixed Reality, gaze, gaze targeting, interaction, design, HoloLens, multimodal 
 ---
 
 # Introducing instinctual interactions
 
-The philosophy of simple, instinctual interactions is woven throughout the Microsoft Mixed Reality (MMR) platform, from hardware to software.
+The philosophy of simple, instinctual interactions is woven throughout the Mixed Reality platform, from hardware to software.
 
-These instinctual interactions utilize all available input technologies, including inside-out tracking, hand tracking, eye tracking, and natural language, in seamless multimodal interaction models. Based on our research, designing and developing multimodals, and not based on single inputs, is critical when creating instinctive experiences.
+These instinctual interactions utilize all the available input technologies, including inside-out tracking, hand tracking, eye tracking, and natural language and come together in the form of our multimodal interaction models. Based on our research, to design and develope instinctive experiences, multimodal is critical - you can't rely on single inputs.
```
Comment (sostel) [inline mixed-reality-docs/interaction-fundamentals.md]: Typo - develope

File: mixed-reality-docs/interaction-fundamentals.md [hunk 2]
```diff
@@ -6,36 +6,62 @@ ms.author: shengkait
...
 
-The Instinctual Interaction models also naturally align across device types.  For example, far interaction on an immersive headset with a 6 degrees of freedom (DoF) controller and far interaction on a HoloLens 2 use the same affordances, patterns, and behaviors.  Not only is this convenient for developers and designers, but it feels natural to end users.
+These instinctual interaction models also naturally align across device types.  For example, far interaction on an immersive headset with a 6 degrees of freedom (6 DoF) controller and far interaction on HoloLens 2 use the same affordances, patterns, and behaviors.  Not only is this convenient for developers and designers, but it feels natural to end users.
```
Comment (sostel) [inline mixed-reality-docs/interaction-fundamentals.md]: Seems like there are some white space before "For example"

File: mixed-reality-docs/interaction-fundamentals.md [hunk 3]
```diff
@@ -6,36 +6,62 @@ ms.author: shengkait
...
 
-
-Lastly, while we recognize that there are thousands of effective, engaging, and magical interactions possible in MR, we have found that intentionally employing a single interaction model end to end in an application is the best way to ensure users are successful and have a great experience.  To that end, we've included three things in this interaction guidance:
+Lastly, while we recognize that there are thousands of effective, engaging, and magical interactions possible in mixed reality, we have found that intentionally employing a single interaction model end to end in an application is the best way to ensure users are successful and have a great experience.  To that end, we've included three things in this interaction guidance:
```
Comment (sostel) [inline mixed-reality-docs/interaction-fundamentals.md]: What did we agree on for "mixed reality" vs. "Mixed Reality". I personally find the latter better.

File: mixed-reality-docs/interaction-fundamentals.md [hunk 4]
```diff
@@ -6,36 +6,62 @@ ms.author: shengkait
...
 * We've structured this guidance around the three primary interaction models and the components and patterns required for each
 * We've included supplemental guidance on other benefits that our platform provides
 * We've included guidance to help select the appropriate interaction model for your scenario
 
 ## Multimodal interaction models
 
-Based on our research and work with customers to date, we've discovered three primary interaction models that suit the majority of Mixed Reality experiences.  
-
-Think of these interaction models as the user's mental model for completing their flows.
+Based on our research and work with customers to-date, we've discovered three primary interaction models that suit the majority of Mixed Reality experiences. Think of these interaction models as the user's mental model for completing a task flow.
```
Comment (sostel) [inline mixed-reality-docs/interaction-fundamentals.md]: Here it's Mixed Reality. Let's agree on a consistent norm.

File: mixed-reality-docs/interaction-fundamentals.md [hunk 5]
```diff
@@ -6,36 +6,62 @@ ms.author: shengkait
...
 
-**Model** | **[Hands and motion controllers](hands-and-tools.md)** | **[Hands free](hands-free.md)** | **[Head-gaze and commit](gaze-and-commit.md)**
-|--------- | --------------| ------------| ---------|
-**Example Scenarios** | 3D spatial experiences, e.g. spatial layout and design, content manipulation, or simulation | Contextual experiences where a user's hands are occupied, e.g. on the-job learning, maintenance| Click-through experiences, e.g. 3D presentations, demos
-**Fit** | Great for new users, coupled wit voice, eye tracking or head gaze. Low learning curve. Consistent UX across hand tracking and 6 DoF controllers. | Some learning required. If hands are unavailable pairs well with voice and natural language | Requires training on HMDs but not on mobile. Best for accessible controllers. Best for HoloLens (1st gen). |
-**Hardware** | HoloLens 2 <br>Immersive headsets | HoloLens 2 <br>HoloLens (1st gen) <br>Immersive headsets | HoloLens 2 <br>Immersive headsets | HoloLens 2 <br>HoloLens (1st gen) <br>Immersive headsets <br>Mobile AR |
+<br>
+<table>
```
Comment (sostel) [inline mixed-reality-docs/interaction-fundamentals.md]: The hardware column made some unfortunate breaks making it more difficult to read Hololens
2
and so on :)

File: mixed-reality-docs/interaction-fundamentals.md (hunk 4 above)
Comment (jessemcculloch) [reply mixed-reality-docs/interaction-fundamentals.md]: per marketing it should be mixed reality unless used as brand

Comment (sostel) [reply mixed-reality-docs/interaction-fundamentals.md]: Cool! Thanks for clarifying, Jesse! :)

**Comment by jessemcculloch** [comment]
@rwinj @sostel - Who is making these changes? Also, at this point there are merge conflicts that reed to be resolved.

thanks, Jesse

**Comment by sostel** [comment]
> @rwinj @sostel - Who is making these changes? Also, at this point there are merge conflicts that reed to be resolved.
> 
> thanks, Jesse

I was just reviewing it. I was assuming that @rwinj would address them at some point?

**Comment by rwinj** [comment]
Sure - I can make sure the lower case mixed reality changes get in there. The article had an extensive update last week so maybe I'll start fresh with these changes vs. sorting out the merge conflicts. Also this article is in the hands of our editor right now (v-Steve DeMont). I can also ask him to include this capitalization change in his edits.

**Comment by rwinj** [comment]
Capital M - Mixed Reality, only appears once in the current live version of the article. I'll get that changed.

**Comment by jessemcculloch** [comment]
Ok, i'm going to close this PR then, since Steve is working on a more up to date version anyways.  Thanks!

**Pull request closed without merging**